src/app/aws.rs

```diff
@@ -183,27 +92,19 @@ pub async fn get_eks_token(cluster_name: &str, role_arn: &Option<String>) -> Res
 mod test {
     use super::*;
 
-    #[test_log::test(tokio::test)]
```

Comment:
I actually want test log here as it sets up logging for the tests: https://docs.rs/test-log/latest/test_log/attr.test.html